tools/mason/MasonModify.chpl

```diff
@@ -28,6 +28,12 @@ proc masonModify(args) throws {
       masonModifyHelp();
       exit();
     }
+    else if args.size == 3{
```

Comment:
Make sure that this is stilling passing all of the modify tests. If the argument size is 3, e.g. ``mason add hdf5`` the else clause in the interface will not be reached and nothing will happen.

Reply:
It seems to be still passing the tests, but only because the tests do a direct call of the modify function and bypass where it goes wrong in MasonModify.chpl, so Ill fix MasonModify and perhaps also right a test that would catch this error if it were to occur again?

Reply:
Still having this issue, just placing this comment for record.